drafts/25-114.txt

```diff
@@ -246,6 +247,9 @@ FPP will not expand tokens in either fixed- or free-form:
     - In character literal constants.
     - In character string edit descriptors (quoted character strings).
 
+FPP will not expand tokens in any alternative syntax for character
+literal constants or edit descriptors that the processor supports.
```

Comment:
If we cannot resolve the above in short order, then in the interests of time I'll suggest reverting to something based on the phrasing Malcolm provided:

```suggestion
If a processor provides an alternative syntax for character literal constants, 
FPP macro expansion shall not take place within such literal constants.
```

If someone very picky wants to stand up and scream that F66 Hollerith edit descriptors did not technically qualify as "literal constants", then we can try to reach a wording they'll agree to for the syntax paper, but this doesn't seem critical to wordsmith for requirements.

Reply:
When he brought up Hollerith, I took that as a sign that he was having to really reach to find stuff to complain about.

Reply:
I don't want to set us up to have to do a 25-114r3